### PR TITLE
Adjust webpack build process to address IE11 polyfill needs.

### LINKED ISSUE #10: Recent MWDK/etc. upgrades broke the widget in IE11/Edge.

### PR DESCRIPTION
**Pull request by FrenjaminBanklin**
Fixes #10.

Added a bunch of polyfill to the webpack config so the built widget runs in poorly engineered outdated browsers.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -9,12 +9,36 @@ const copy = widgetWebpack.getDefaultCopyList()
...
 
 entries['assets/js/creator.js'] = [
+	'core-js/es6/array',
+	'core-js/fn/array/map',
+	'core-js/es6/symbol',
+	'core-js/es6/promise',
+	'core-js/fn/set',
+	'core-js/fn/object/assign',
+	'core-js/fn/string/includes',
+	'core-js/web/dom-collections',
```
Comment (iturgeon) [inline webpack.config.js]: Are all of these needed in all cases? ideally we'd only include the shims for what's used.

File: webpack.config.js [hunk 2]
```diff
@@ -36,7 +60,13 @@ const babelLoaderRule = {
...
 		loader: 'babel-loader',
 		options: {
-			presets: ['env']
+			presets: [
+				'es2015',
+				['env', {
+					targets: { browsers: ["last 2 versions", "ie >= 11"]},
+					debug: true
+				}]
+			]
```
Comment (iturgeon) [inline webpack.config.js]: Note that there are separate but different settings for babel in package.json attempt to consolidate?

Comment (FrenjaminBanklin) [reply webpack.config.js]: Moved preset options from the webpack config into package.json. Seems to work.

File: webpack.config.js (hunk 1 above)
Comment (FrenjaminBanklin) [reply webpack.config.js]: Turns out only one shim was actually necessary, though the rest didn't seem to be hurting anything.

**Pull request merged**